diff -urN -x .build -x .inst -x .sinst -x configure -x 'Makefile.in*' -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x 'intltool*' -x autom4te.cache -x '*compile' -x '*stamp' -x '*html' -x '*sgml' -x '*tmpl' -x '*txt' -x '*xml' gcursor-0.061-orig/CYGWIN-PATCHES/README gcursor-0.061/CYGWIN-PATCHES/README
--- gcursor-0.061-orig/CYGWIN-PATCHES/README	1969-12-31 18:00:00.000000000 -0600
+++ gcursor-0.061/CYGWIN-PATCHES/README	2006-01-04 14:26:42.375000000 -0600
@@ -0,0 +1,73 @@
+gcursor
+------------------------------------------
+GNOME mouse cursor theme selector.
+
+Runtime requirements:
+  cygwin-1.5.18-1
+  GConf2-2.12.1-1
+  gtk2-x11-runtime-2.6.10-1
+  libglade2-2.5.1-1
+  libgnome2-2.12.0.1-1
+  libgnomeui2-2.12.0-1
+  xorg-x11-bin-dlls-6.8.2.0-1
+
+Build requirements:
+(besides corresponding -devel packages)
+  cygwin-1.5.18-1
+  autoconf2.5-2.59-3
+  automake1.9-1.9.6-1
+  binutils-20050610-1
+  gawk-3.1.5-1
+  gcc-core-3.4.4-1
+  make-3.80-1
+  pkgconfig-0.17.2-2
+  sed-4.1.4-1
+
+Canonical website:
+  http://qballcow.nl/index.php?name=gcursor
+
+Canonical download:
+  http://download.qballcow.nl/programs/gcursor/
+
+-------------------------------------------
+
+Build instructions:
+  unpack gcursor-0.061-X-src.tar.bz2
+    if you use setup to install this src package, it will be
+	 unpacked under /usr/src automatically
+  cd /usr/src
+  ./gcursor-0.061-X.sh all
+
+This will create:
+  /usr/src/gcursor-0.061-X-src.tar.bz2
+  /usr/src/gcursor-0.061-X.tar.bz2
+
+-------------------------------------------
+
+Files included in the binary package:
+
+  /usr/bin/gcursor.exe
+  /usr/share/applications/gcursor.desktop
+  /usr/share/doc/Cygwin/gcursor-0.061.README
+  /usr/share/doc/gcursor-0.061/AUTHORS
+  /usr/share/doc/gcursor-0.061/ChangeLog
+  /usr/share/doc/gcursor-0.061/COPYING
+  /usr/share/doc/gcursor-0.061/INSTALL
+  /usr/share/doc/gcursor-0.061/NEWS
+  /usr/share/doc/gcursor-0.061/README
+  /usr/share/doc/gcursor-0.061/TODO
+  /usr/share/gcursor/gcursor.glade
+  /usr/share/locale/de/LC_MESSAGES/gcursor.mo
+  /usr/share/locale/es/LC_MESSAGES/gcursor.mo
+  /usr/share/locale/sv/LC_MESSAGES/gcursor.mo
+
+------------------
+
+Port Notes:
+
+----- version 0.061-1 -----
+Initial release for Cygwin.
+
+
+Cygwin port maintained by:
+http://cygwinports.dotsrc.org/
diff -urN -x .build -x .inst -x .sinst -x configure -x 'Makefile.in*' -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x 'intltool*' -x autom4te.cache -x '*compile' -x '*stamp' -x '*html' -x '*sgml' -x '*tmpl' -x '*txt' -x '*xml' gcursor-0.061-orig/CYGWIN-PATCHES/setup.hint gcursor-0.061/CYGWIN-PATCHES/setup.hint
--- gcursor-0.061-orig/CYGWIN-PATCHES/setup.hint	1969-12-31 18:00:00.000000000 -0600
+++ gcursor-0.061/CYGWIN-PATCHES/setup.hint	2006-01-04 14:26:42.375000000 -0600
@@ -0,0 +1,4 @@
+category: Gnome X11
+requires: cygwin GConf2 gtk2-x11-runtime libglade2 libgnome2 libgnomeui2 xorg-x11-bin-dlls xorg-x11-base
+sdesc: "GNOME mouse cursor theme selector"
+ldesc: "GNOME mouse cursor theme selector"
diff -urN -x .build -x .inst -x .sinst -x configure -x 'Makefile.in*' -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x 'intltool*' -x autom4te.cache -x '*compile' -x '*stamp' -x '*html' -x '*sgml' -x '*tmpl' -x '*txt' -x '*xml' gcursor-0.061-orig/src/Makefile.am gcursor-0.061/src/Makefile.am
--- gcursor-0.061-orig/src/Makefile.am	2004-04-06 16:02:40.000000000 -0500
+++ gcursor-0.061/src/Makefile.am	2006-01-04 14:17:37.546875000 -0600
@@ -5,6 +5,7 @@
 		  gcursor.c	\
 		  $(NULL)
 gcursor_LDADD	= $(GCURSOR_LIBS)
+gcursor_LDFLAGS = -Wl,--export-all-symbols
 
 AM_CFLAGS	= $(GCURSOR_CFLAGS) -DGLADE_PATH=\""$(datadir)/gcursor/"\"
 
